spec/acceptance/nodesets/ec2/image_templates.yaml

```diff
@@ -27,6 +27,16 @@ AMI:
     :image:
       :aio: ami-fe408091
     :region: eu-central-1
+  # Debian 8.7 (HVM),
```

Comment:
This modification can be removed

Reply:
Line 30-34, ok.
Lines 35-39 too?

Reply:
please do not modify this file with the PR.